src/LearnositySdk/Request/Remote.php

```diff
@@ -54,6 +54,50 @@ public function post($url, $data = array(), $options = array())
     }
 
     /**
+     * Determine whether a header is an 'Expect' header.
+     * @param string $header
+     * @return boolean
+     */
+    private function isExpectHeader($header)
+    {
+        if (strpos(strtolower($header), 'expect:') === 0) {
+            return True;
+        }
+        return False;
```

Comment:
In this particular case I disagree with that: I'd prefer it to err toward clarify than terseness in code that clients may read to debug or understand.